### PR TITLE
[TEVA-4078] Update long form index page content

### PR DESCRIPTION
**Pull request by josephhull676**
## Jira ticket URL

https://dfedigital.atlassian.net/browse/TEVA-4078

### CODE REVIEW

**Review by robyoung91 — APPROVED** [review]

File: config/locales/en.yml [hunk 1]
```diff
@@ -291,13 +291,18 @@ en:
   posts:
     index:
       get-help-hiring:
-        title: School hiring guides
```
Comment (cesidio) [inline config/locales/en.yml:294]: why did you remove this line?
